cmd/root.go

```diff
@@ -81,25 +84,25 @@ func Execute() {
 	}
 }
 func startScanner(namespace string, clusterWide bool, scanner Scanner) error {
-	if clusterWide {
-		if err := scanner.ScanClusterWideResources(); err != nil {
-			return err
-		}
+	if clusterWide && namespace != "" {
+		log.Fatal().Msg("Cannot scan cluster wide and only a namespace at the same time")
```

Comment:
Why are these operation mutually exclusive? At first glance, I cannot see any technical reason to block this.

Reply:
At some point we may have stated that we only wanted to run one or the other, hence the implementation. I can't find it in the RFCs, so it must have been in an impromptu talk. It comes from the epic task list too.

IMHO there's no problem with it, we want to scan all ns and clusterwide as default anyways.